persistence/src/commonMain/kotlin/com/wire/kalium/persistence/backup/BackupImporter.kt

```diff
@@ -0,0 +1,57 @@
+package com.wire.kalium.persistence.backup
+
+import app.cash.sqldelight.db.SqlDriver
+
+interface BackupImporter {
+    suspend fun importFromFile(filePath: String)
+}
+
+class BackupImporterImpl(private val sqlDriver: SqlDriver) : BackupImporter {
+
+    override suspend fun importFromFile(filePath: String) {
```

Comment:
I would execute this with `wrapStorageRequest` so that we don't crash the app on failed operations over the DB.